linera-views/src/dynamo_db.rs

```diff
@@ -69,7 +70,7 @@ const MAX_BATCH_WRITE_ITEM_BYTES: usize = 16777216;
 
 /// Fundamental constant of DynamoDB: The maximum number of simultaneous connections is 50.
 /// See https://stackoverflow.com/questions/13128613/amazon-dynamo-db-max-client-connections
```

Comment:
Is this reference saying what we think it says?

Reply:
This page refers to the page https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/ServiceQuotas.html
which contains the following text: "DynamoDB Import from Amazon S3 can support up to 50 concurrent import jobs with a total import source object size of 15TB at a time in us-east-1, us-west-2, and eu-west-1 regions"

But if the import jobs are overlapping, then likely there is no limit and then it is "load balanced to infinity".

That being said, it is possible that we have at the same time:
* The limit of 50 that I wrote do not really exist.
* There is a need for a limit to a specific.
* There is also a need for semaphores for the number of concurrent dynamoDb tests that can run at the same time just like for rocksDB.

Reply:
This is about "import jobs from Amazon S3" and seems a bit unrelated. Anyways, I think we all agree that the number of connections can/should be limited during tests. Maybe just make sure that this is configurable

Reply:
Ok, you are right. It is significant coding, but we want to make it a configurable parameter.
I will finish this tomorrow.

Reply:
Also, I would still like to understand what's happening under the hood during the tests with `dynamodb-local`.

Reply:
I found this: https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.UsageNotes.html

Reply:
I did not know about this document, I will look in detail at the possible options since I need to crack down on that bug.
But I think the limitation of the number of transactions is of more general interest.